windows/ReactNativePicker/ReactPackageProvider.cpp

```diff
@@ -0,0 +1,17 @@
+﻿#include "pch.h"
+#include "ReactPackageProvider.h"
+#if __has_include("ReactPackageProvider.g.cpp")
+#include "ReactPackageProvider.g.cpp"
```

Comment:
curious about when this file might/might not exist? what generates this?

Reply:
This was copied from the old PickerViewManager. I guess the file should always be there in current setup, removed the if statement.